_posts/2020-06-12-release-2.7.0.md

```diff
@@ -0,0 +1,20 @@
+---
+layout: default
+title: Release v2.7.0
+tags: ["v2.7.0", "changelog"]
+excerpt_separator: <!--more-->
+---
+
+As of 12/06 2020 the new version 2.7.0 is out
+<!--more-->
+
+## Standardization & Menas Improvements
+
+- [#1235]({{ site.github.issues_url }}/1235) [BinaryType]({{ site.baseurl }}/docs/2.0.0/usage/schema#binary) is now supported 
+both in Standardization/Conformance and in menas UI. Mind the [`encoding`]({{ site.baseurl }}/docs/2.0.0/usage/schema#encoding) 
```

Comment:
```suggestion
both in Standardization/Conformance and in Menass UI. Mind the [encoding]({{ site.baseurl }}/docs/2.0.0/usage/schema#encoding) 
```
The encoding in backquotes will not be as a link by anyone. I would have it as a plain text.

Link in backquotes:
![image](https://user-images.githubusercontent.com/17467768/84592383-a86e5180-ae45-11ea-9cb6-545bbd5f36b6.png)

Reply:
I dunno. I find the formatting more important then noticing the link. After all the BinaryType as a whole is linked visibly. This is just something for the observant ones going directly. Others' opinons?

Reply:
@Zejnilovic: `Menass` -> `Menas`?

Reply:
> @Zejnilovic: `Menass` -> `Menas`?

I might have made a mistake there

Reply:
In any case it's Menas now, thanks